src/Sarif/FileRegionsCache.cs

```diff
@@ -234,7 +234,7 @@ private void PopulateEndColumn(NewLineIndex lineIndex, Region region, string fil
             }
         }
 
-        private static void PopulateCharOffset(NewLineIndex lineIndex, Region region, string fileText)
```

Comment:
>string fileText [](start = 86, length = 15)

FxCop notices unused parameters. I scrubbed some of these issues in order to generate 'absent' messages in the project file.

Reply:
@lgolding @Evmaus-MS @EasyRhin0